tech/languages/python/fastapi-installation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: FastApi
+subsection: python
+order: 5
+---
+
+# FastApi
+
+[FastApi](https://fastapi.tiangolo.com/) is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+
+## Installation of FastApi in a virtual environment
+
+It is always useful to install FastApi as a third-party package inside a *virtual environemnt*.
+This will keep your project separate from your system, giving you more freedom in choosing additional libraries & their versions.And it's easy collaborating with different operating system.
+
+Let's create a new project & a virtual environement
+Open the terminal. Then, create a new folder 'my_project', open it, and create a virtual environment called 'project_env' or whatever you like.
+
+```bash
+$ mkdir my_project && cd my_project
+$ virtualenv project_env
+```
+To work in the virtual environment, you have to activate it.
+
+```bash
+$ source project-env/bin/activate
+```
+
+In an active the virtual environment (with the name `(project_env)` included in your command line prompt), you can install fastapi.
```

Comment:
```suggestion
Now that we have an active virtual environment (this can be identified by having the `(project_env)` in the command line prompt), install FastApi and a server using `pip`.
```